components/generate/GenerateModeSelector.tsx

```diff
@@ -41,17 +33,14 @@ function GenerateModeSelector({ selectedType }: Props) {
               )}
               title="Create with Complete Assets"
               description="Ticket will be created with your uploaded assets such as image or video. You can also customize the quantity of each assets to be distributed to your customers."
-              containerStyles={{ backgroundColor: orange[400], width: "100%" }}
+              containerStyles={{ backgroundColor: orange[400] }}
             />
           </div>
 
           <div
             onClick={() => {
               router.push({
-                pathname: "ticket/layer",
-                query: {
-                  ticketType: selectedType
-                }
+                pathname: "ticket/layer"
               })
             }}
```

Comment:
If you don't wish to push with any data you can stick with `router.push("ticket/layer")`

Reply:
this will be removed after I finish with Complete form